building/build-debs/homeworld-etcd-metrics-exporter/inner-build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e -u
+
+rm -rf go
+tar -xf go-bin-1.8.3.tgz go/
+export GOROOT="$(pwd)/go/"
+export PATH="$PATH:$GOROOT/bin"
+
+if [ "$(go version 2>/dev/null)" != "go version go1.8.3 linux/amd64" ]
+then
+	echo "go version mismatch! expected 1.8.3" 1>&2
+	go version 1>&2
+	exit 1
+fi
+
+PROMETHEUS_VER="2.0.0"
+PROMETHEUS_CLIENT_GOLANG_VER="0.9.0-pre1"
+
+rm -f src/github.com
+rm -rf "prometheus-${PROMETHEUS_VER}"
+tar -xf "prometheus-${PROMETHEUS_VER}.tar.xz" "prometheus-${PROMETHEUS_VER}/vendor"
+tar -xf "prometheus-client_golang-${PROMETHEUS_CLIENT_GOLANG_VER}.tar.xz" "client_golang-${PROMETHEUS_CLIENT_GOLANG_VER}"
+rm -rf "prometheus-${PROMETHEUS_VER}/vendor/github.com/prometheus/client_golang/"
+mv "client_golang-${PROMETHEUS_CLIENT_GOLANG_VER}" -T "prometheus-${PROMETHEUS_VER}/vendor/github.com/prometheus/client_golang/"
```

Comment:
It looks like -T is unnecessary here if you remove the trailing slash, especially since you just rm'ed it.

Reply:
I include things like `-T` here because they, in a number of cases, prevent errors, and they don't cost much to include.